portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/login/Login.testcase

```diff
@@ -490,4 +490,37 @@ definition {
 		}
 	}
 
+	@description = "Add a new admin user, the user creates a new page and web content, and adds a web content widget to the page"
+    @priority = 5
+	test AdminAddPageAndWebContent{
+		User.loginPG();
+
+		JSONUser.addUserWithRole(
+		userEmailAddress = "test2@liferay.com",
```

Comment:
Your former doesn't correct, please execute the source format before pushing it.